cargo-concordium/src/main.rs

```diff
@@ -471,6 +477,34 @@ fn print_contract_schema_v1(
     }
 }
 
+// TODO: Consider refactoring to reuse v1 version.
```

Comment:
Let's not have this TODO.

This code is fine and understandable. While there is a bit of duplication, introducing an abstraction to handle the two will also introduce more code. 

Once we have the fourth schema version then perhaps we can refactor :)